src/workers/postUpdated.ts

```diff
@@ -82,8 +92,10 @@ const worker: Worker = {
             : data?.extra?.creator_twitter;
 
         const authorId = await findAuthor(entityManager, creatorTwitter);
-        const title = data?.title || databasePost.title;
-        const becomesVisible = !!title?.length;
+
+        const { private: privacy } = await entityManager
```

Comment:
Let's select only the relevant field then. It's a minor optimization but I want us to get used to selecting relevant fields when possible :)